Model/Sales/Repository/PackageRepository.php

```diff
@@ -94,39 +112,48 @@ public function productWithoutDeliveryOptions(array $products)
     }
 
     /**
+     * @param $product
+     * @param $mailBox
+     *
      * @return bool
      */
-    public function fitInMailbox(): bool
+    public function fitInMailbox($product, $mailBox): bool
```

Comment:
type hints

Reply:
en mailbox is 1 woord dus die hoef je niet te camelCasen

Reply:
done